src/Backoffice/Employee/App/Controllers/StoreEmployeeController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Employee\App\Controllers;
+
+use Illuminate\Http\JsonResponse;
+use Lightit\Backoffice\Employee\App\Requests\StoreEmployeeRequest;
+use Lightit\Backoffice\Employee\App\Transformers\EmployeeTransformer;
+use Lightit\Backoffice\Employee\Domain\Actions\StoreEmployeeAction;
+
+class StoreEmployeeController
+{
+    public function __invoke(
+        StoreEmployeeRequest $request,
+        StoreEmployeeAction $action,
```

Comment:
The same, maybe it's not a big deal here, but when you have 2 or 3 actions, it's better to have proper names
```suggestion
        StoreEmployeeAction $storeEmployeeAction,
```